Training/configs/scaling_params_vDisTauTag_v1.json

```diff
@@ -0,0 +1,276 @@
+{
+    "PfCand": {
+        "pfCand_valid": {
+            "global": {
+                "mean": 0,
+                "std": 1,
+                "lim_min": "-inf",
+                "lim_max": "inf",
+                "sqmean": null,
+                "num": null
+            }
+        },
+        "pfCand_pt": {
+            "global": {
+                "mean": 50.0,
+                "std": 50.0,
+                "lim_min": -1.0,
+                "lim_max": 1.0,
+                "sqmean": null,
+                "num": null
+            }
+        },
+        "pfCand_eta": {
+            "global": {
+                "mean": 0.0,
+                "std": 3.0,
+                "lim_min": -1.0,
+                "lim_max": 1.0,
+                "sqmean": null,
+                "num": null
+            }
+        },
+        "pfCand_phi": {
+            "global": {
+                "mean": 0.0,
+                "std": 3.141592653589793,
+                "lim_min": -1.0,
+                "lim_max": 1.0,
+                "sqmean": null,
+                "num": null
+            }
+        },
+        "pfCand_mass": {
```

Comment:
Hi @kandrosov I am now in process of testing it, also in my plan to test if it is better to use log(pt) and log(E). the ranges of pt (or log(pt)) also will be changed.

Also, to be consistent I found out that it is better to use dxy_sig, and chi2/ndf and additionally adding sv and lostTracks, I will present soon this features tuning and then merge the final model already in the end when I hypertune architecture, but I propose to merge this as it is now, because this is how `model_v1` was trained that is currently used by me and other stau group